packages/builder/src/pages/builder/app/[application]/design/[screenId]/[componentId]/_components/Component/ComponentSettingsSection.svelte

```diff
@@ -17,13 +17,23 @@
   export let onUpdateSetting
   export let showSectionTitle = true
   export let showInstanceName = true
+  export let tag
 
-  $: sections = getSections(componentInstance, componentDefinition, isScreen)
+  $: sections = getSections(
+    componentInstance,
+    componentDefinition,
+    isScreen,
+    tag
+  )
 
   const getSections = (instance, definition, isScreen) => {
```

Comment:
Although I think this will work since the reactive statement will re-run whenever `tag` changes, I think we should still have `tag` listed as a param here.